raster/src/main/scala/geotrellis/raster/histogram/Histogram.scala

```diff
@@ -136,12 +134,12 @@ abstract trait Histogram extends Serializable {
       var needMedian = true
       val limit = getTotalCount() / 2
 
-      var i = 0
       val len = values.length
 
-      while (i < len) {
+      cfor(0)(_ < len, _ + 1) { i =>
```

Comment:
Sometimes a while loop is faster than a cfor. Don't need to change these. We'll keep these in, but then benchmark the old vs new histogram and maybe change back, just wanted to give a heads up